gensrc/thrift/InternalService.thrift

```diff
@@ -90,7 +90,7 @@ struct TQueryOptions {
   1: optional bool abort_on_error = 0
   2: optional i32 max_errors = 0
   3: optional bool disable_codegen = 1
-  4: optional i32 batch_size = 0
+  4: optional i32 chunk_size = 0
```

Comment:
shouldn't change the name, which has a compatible issue

Reply:
[m](https://stackoverflow.com/questions/52882370/is-safe-to-rename-a-field-in-thrift-idl) rename field is safe @kangkaisen

Reply:
https://stackoverflow.com/questions/52882370/is-safe-to-rename-a-field-in-thrift-idl

Reply:
I've revoked to use batch_size, actually batch_size used by some users for some time.